### DISCUSSION TITLE
Option for running watchtower without docker

### DISCUSSION THREAD

**Discussion by MauriceNino** [containrrr/watchtower#1301] [Feature Requests] [answered]
Hi there! I am in the progress of creating an alternative to Portainer and Yacht and would like to integrate watchtower for automatic image updates.
For that, I planned on bundling Watchtower directly into my image to avoid spinning up any containers on the fly (like Yacht is doing it AFAIK).

1. Is there any documented way to start up a prod instance without a docker container?
2. Are there any plans on building out the HTTP Api to target specific containers?

Also, would you need any help with that (PR)? I have never written anything in Go, but I can try.

> Sidenote: Users would be exposed to the name "Watchtower" and a link to this project, as there will be a configuration section for it.

**Comment by piksel** [comment, marked as answer]
> Is there any documented way to start up a prod instance without a docker container?

The releases simply contain a single binary (pick the one for your platform). Just run it the same way as the container, with either `WATCHTOWER_` environment variables, or arguments from the docs.

> Are there any plans on building out the HTTP Api to target specific containers?

There is a PR that adds filtering by image name to the API, and this should be extendend to support all the current filters (including container names/IDs). Ref #1289

**Reply by MauriceNino** [reply]
Great, thank you for the answers. Hopefully #1289 gets picked up again.
